### ISSUE TITLE
Issue with stream

### ISSUE THREAD

**Issue by brookeqt** [didlly/grank#37]
If it detects you don't have a mouse and keyboard, it buys a keyboard only, resulting in it buying another keyboard because the mouse is missing every time it uses the command

**Comment by didlly** [comment]
I'll fix this when I get back to my computer later today.

**Comment by brookeqt** [comment]
Also will this be updated to slash commands

**Comment by didlly** [comment]
> Also will this be updated to slash commands

Yes.

**Comment by didlly** [comment]
Okay, I've just realised that while this has been fixed, there are still ways to improve the stream command. I'll improve it when I get on my PC.
